src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -218,11 +218,14 @@ export function PriceChart({ width, height, token }: PriceChartProps) {
   const [delta, arrow] = getDelta(startingPrice.value, displayPrice.value)
   const crosshairEdgeMax = width * 0.85
   const crosshairAtEdge = !!crosshair && crosshair > crosshairEdgeMax
+  const priceNumDecimalPlaces = displayPrice.value < 1.05 ? 4 : 2
 
   return (
     <>
       <ChartHeader>
-        <TokenPrice>${displayPrice.value.toFixed(2)}</TokenPrice>
+        <TokenPrice>
+          {displayPrice.value < 0.0001 ? '$<0.0001' : displayPrice.value.toFixed(priceNumDecimalPlaces)}
```

Comment:
could we use `formatDollarAmount`?

Reply:
also noticed this is .0001 and the other is .001, not sure if intended

Reply:
i was pondering, but i think they wanted to show up to four decimal points when it's < $1.05 and formatDollarAmount does $<0.001 (three decimal points) -- wdyt

Reply:
i think you should ask design to standardize and then use that util

Reply:
checkout formatToDecimal here: interface/src/components/AmplitudeAnalytics/utils.ts. 

also I think 6 significant figures for smaller values makes sense over 4 (it would be consistent with other parts of the codebase).

Reply:
I agree! thanks Lynn